### PR TITLE
allows you to heal organs using organ manipulation with bruise packs or ointment (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr https://github.com/Skyrat-SS13/Skyrat13/pull/773
"Adds a "heal" option for organ manipulation. By using 3 stacks of ointment or bruise packs on the organ manipulation step, you can heal all organs in the targeted zone completely."
#### Changelog
// maintainer edit: fixed changelog
:cl:  BobJoga
rscadd: ports the ability to heal organs with bruise / ointment
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
what is this reverting?

**Comment by aaqe** [comment]
This doesn't seem like real medicine at all.

**Comment by boodaliboo** [comment]
> This doesn't seem like real medicine at all.

and? ss13 medicine acts nothing like real medicine

**Comment by ToGWtF** [comment]
>Heal all organs completely.

Dont. Make it heal a bit, or even quite a lot of damage, but don't let it literally cure heart disease with a few  bandaids.

**Comment by aaqe** [comment]
I mean to say this isn't future tech and it's stupid.

**Review by JamieD1 — APPROVED** [review]

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Erm... Why did you duplicate the proc at line 80 instead of changing it....

**Comment by boodaliboo** [comment]
> Erm... Why did you duplicate the proc at line 80 instead of changing it....

im not sure what to do if i do remove it

**Comment by JamieD1** [comment]
Sent changes in discord

File: code/modules/surgery/organ_manipulation.dm [hunk 1]
```diff
@@ -154,4 +172,10 @@
 			display_results(user, target, "<span class='notice'>You can't extract anything from [target]'s [parse_zone(target_zone)]!</span>",
 				"[user] can't seem to extract anything from [target]'s [parse_zone(target_zone)]!",
 				"[user] can't seem to extract anything from [target]'s [parse_zone(target_zone)]!")
+
+	else if(current_type == "heal")
+		for(var/obj/item/organ/O in target.getorganszone(target_zone))
+			display_results(user, target, "<span class='notice'>You succesfully heal [target]'s [parse_zone(target_zone)]'s organs.</span>",
```
Comment (Ktlwjec1) [inline code/modules/surgery/organ_manipulation.dm:178]: successfully

File: code/modules/surgery/organ_manipulation.dm [hunk 2]
```diff
@@ -126,6 +126,24 @@
...
+		if(istype(tool, /obj/item/stack/medical/bruise_pack))
+			var/obj/item/stack/medical/M = tool
+			if(M.use(3))
+				current_type = "heal"
+				var/list/organlist = target.getorganszone(target_zone)
+				if(!organlist.len)
+					display_results(user, target, "<span class='notice'>There are no organs in the [parse_zone(target_zone)]!</span>",
+					"[user] begins to heal [target]'s [parse_zone(target_zone)] organs, only to be met with no organs at all.",
+					"[user] begins to heal [target]'s [parse_zone(target_zone)] organs, only to be met with no organs at all.")
+					return -1
```
Comment (alexkar598) [inline code/modules/surgery/organ_manipulation.dm:139]: This should be before the M.use(3) check

**Review by alexkar598 — CHANGES_REQUESTED** [review]
I have no words, this doesnt heal the person at all

**Comment by JamieD1** [comment]
@boodaliboo Update this or I close it

**Comment by BobJoga** [comment]
While i get giddy about my work being ported to other codebases, this PR does not port along the bugfixes and improvements i did to this code

**Pull request closed without merging**